src/blocks/small-search.css

```diff
@@ -0,0 +1,45 @@
+.small-search__input {
+  font-weight: 400;
+  font-size: 14px;
+  margin: 0 auto;
+  width: 100%;
+  text-indent: 33px;
+  border: none;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11;
+  background-position: 13 14;
```

Comment:
The 'background-position' property is missing units. Both values should be specified in a valid CSS unit like 'px', 'em', etc.